### ISSUE TITLE
Compatibility issues with EnderIO

### ISSUE THREAD

**Issue by ShaCraft** [McJty/DeepResonance#111]
Compatibility issues with EnderIO. causes Server crash when teleporting to overworld (nether, Aroma, ect).  

My tracking this bug  info at EnderIO. 
https://github.com/SleepyTrousers/EnderIO/issues/3626

Sha

**Comment by McJty** [comment]
Please update Deep Resonance and ElecCore. You're using an outdated version

**Comment by ShaCraft** [comment]
I have that log was from the start of the issue. Read the entire report please.

read test #3
""
Test 1: IO/core only. No crash (10 portals in new chunks with 150+ teleports).
Test 2: testing update IO/Core with full pack (and updating the rabbit hole of mods):
Updated IO/Core, Forge and JEI: same results on crash.
Test 3: Updated All mods; same crash
Test 4: starting to add/remove mods leaving IO/core to see if i can track down the mod conflict. 
""

**Comment by Elec332** [comment]
If it crashes with the lastes version of both DR & ElecCore, please upload the entire game log.

**Comment by thwafflemymx** [comment]
Is it intentional that there is consistant liquid loss when using the enderio fluid conduits i am really confused(eg i had 10B in a tank and transfered it to recieve 6B)

**Comment by McJty** [comment]
No but are you sure the liquid isn't lost due to the laser? Do you have a laser firing on the tank?

**Comment by thwafflemymx** [comment]
No, it simply tranfers the 100q 85p 10s 10e liquid from one tank to another abot 20blocks worth of pipe via pressurized fluid conduit.it does this even when broken replaced, other tanks.used other liq etc.

**Comment by thwafflemymx** [comment]
Basicaly in any confoguration or form. I cant give you the exact version now but it did update on aug 30th, ill send the versions tomorrow when i have access to the computer

**Comment by thwafflemymx** [comment]
By the way i have noticed thw issue coming and going in previous versions as well, but i just asumed it was intentional because it is always abot 40% of the liquid that is lost(i didnt have many cases to examine but i am pretty sure that is the case

**Comment by thwafflemymx** [comment]
Also i abologize for awful spelling and grammar, but the screen of my phone is barely functional

**Comment by thwafflemymx** [comment]
Also correction not aug the 30th but aug 29 
stupid date autocorrect...

**Comment by thwafflemymx** [comment]
Alright i think i have found ehy that is happening so here goes. As we all know the EIO conduits have  4 modes: Force push, force pull, both , disabled. And the DR tanks have 3 modes: Force pull, force push, disabled. Neither the tank nor the conduit have what would be the equivalent yo TE4 default mode, where the conduit is connected, but it doesnt push or pull, but insthead relies on the tank "telling" it what to do. I have tested and the cause of losing part of the liquid is because the EIO pressurised fluid conduit tries to pull 100mb(default extract rate) and the tank tries to push 100mb in the same tick(doesnt occour every tick but still jappens a lot). So while i think it would be impossible to change the code of either of the mods, there is a simple solution: Simply tell the DR tank to push into a EIO tank and then pull with a conduit, and do vice versa on the recieving end. The versions of theods are irrelevant since i tested on a LOT of versions of either mod, yielding the same result. I really hope i was clear in trying to ecplain what is causing the issue, english isn't my native language. Thank you for the attention anyway MCJty

**Comment by HenryLoenwind** [comment]
So what makes the liquid getting lost? The conduit shouldn't pull nor accept if it has no place to put it. With our tanks combining conduit pull and tank push works fine.

**Comment by McJty** [comment]
yes I'm also curious about how this could happen. And even if it occurs in the same tick then still there should normally not be a loss as the conduit would simply get double the amount of liquid then. Right?

**Comment by HenryLoenwind** [comment]
Exactly. The conduit would either take it (if there was space), do a "push through" (if there's space in an adjacent target) or reject the amount it cannot take.

**Comment by thwafflemymx** [comment]
... Put a tank and a purifier next to itm put a valve below said tank with the 85p config. Put a tank below the valve. Set one side of the tank to push. Connect it to conduit with extract- always active. Connect to other tank which will accept the liquid. Enjoy the result

**Comment by thwafflemymx** [comment]
I really cant explain it more than i did in my long post. Now you can rescreate the same bug. I really am not that good w programming
